testsuite/tests/typing-layouts-untagged-immediate/alloc.ml

```diff
@@ -1,7 +1,7 @@
 (* TEST
- include stdlib_beta;
+ include stdlib_stable;
  flambda2;
- flags = "-extension small_numbers_beta -extension layouts_beta";
+ flags = "-extension-universe beta";
```

Comment:
`-extension-universe stable` I think?  Or you can just leave it off now, since that's the default.